slides/en/index.html

```diff
@@ -320,7 +369,9 @@ <h2>Data: scopes &amp; life cycles</h2>
             </div>
 
             <aside class="notes">
-              Forms data are for example used to store the result of a REST API call (to get BDM value) in order to display it in a form.
+			  1.30<br /><br />
+			  BDM: complex type with fields, dedicated DB with automatic administration, accessible by API<br />
+              Form: used to store the result of a REST API call (like getting BDM data)
```

Comment:
I think the fact that form variable can be used to display the data after storing the result  of a REST API call is useful.

Reply:
Has not been removed: it appears in the same session with slightly different wording.